data/json/monsters/zed-pupating.json

```diff
@@ -119,7 +119,7 @@
     "armor_cut": 10,
     "armor_bullet": 10,
     "armor_elec": 6,
-    "vision_day": 83,
+    "vision_day": 40,
```

Comment:
```suggestion
```
Its better to delete it because 40 is default game value (and game cant accept it)

Reply:
Excellent point. The game will not even load with `40` here:

![image](https://user-images.githubusercontent.com/238395/151794787-c5863c89-27b9-4920-b881-5ab4196fb4b1.png)

Reply:
> Its better to delete it because 40 is default game value (and game cant accept it)

Huge thanks! I changed it for both the decoy pulpating hulk and the "real" pupating hulk since I believe just removing line 122 would throw the same error for the other variant.